examples/prom/nonlinear_elasticity_global_rom.cpp

```diff
@@ -61,6 +60,13 @@
 //      Relative error of ROM position (x) at t_final: 5 is 0.00188458
 //      Relative error of ROM velocity (v) at t_final: 5 is 0.978726
 //
+// Online phase with EQP hyper-reduction
+//      ./nonlinear_elasticity_global_rom -online -dt 0.01 -tf 5.0 -s 14 -vs 100 -eqp -ns 2 -ntw 50  -rvdim 40 -rxdim 10 -hdim 1  -sc 1.00
+// Output message:
+//      Elapsed time for time integration loop 82.0641
```

Comment:
This timing is less than 1 second for me, so this must be an old result. It would be helpful to add timings for the FOM run, so that speedup can be determined. Currently, timings are only recorded for online runs. It is inconvenient to rerun everything and check the timing, but that has to be done for consistency.